### ISSUE TITLE
No matching clause error

### ISSUE THREAD

**Issue by teawaterwire** [district0x/re-frame-web3-fx#4]
I get a `Uncaught Error: No matching clause: 0x01` when `on-tx-receipt` is reached.
The status in the receipt is `[:status "0x01"]` and it's not present in:
```
(condp #(contains? %1 %2) (:status receipt)
              #{"0x0" 0} (when on-tx-error
                           (dispatch (conj (vec on-tx-error) receipt)))
              #{"0x1" 1} (when on-tx-success
                           (dispatch (conj (vec on-tx-success) receipt))))
```
But maybe I'm missing something? Or it has to do with my RPC (Ganache) ?
Thanks 🙏

**Comment by cmal** [comment]
Similar problem. I've got nothing when `on-tx-receipt` or `on-tx-success` is reached.
The `:on-tx-hash` is triggered.
But the `:on-tx-success` or `:on-tx-receipt` are not triggered.

**Comment by madvas** [comment]
hmm interesting, I've never encountered this. What's your Ganache version?

**Comment by teawaterwire** [comment]
My Ganache version is `1.1.0-beta.0 `

**Comment by madvas** [comment]
That's outdated, isn't it? https://github.com/trufflesuite/ganache-cli/releases

**Comment by teawaterwire** [comment]
I'm using the Ganache app (not the cli) but actually I found out I don't have this error anymore with `1.0.2` 🎉 

Btw, thanks for the library it's very useful 👍

**Comment by cmal** [comment]
I am using ganache-cli 6.1.0 and find the same "0x01" error.

**Comment by madvas** [comment]
This issue was fixed in latest release (1.0.4)
